global.json

```diff
@@ -0,0 +1,8 @@
+{
+  "tools": {
+    "dotnet": "3.0.100-preview-009750"
+  },
+  "msbuild-sdks": {
```

Comment:
Make sure you **do** have SDK here. Our build just broke when they made some changes to the agents (because we missed specifying it here).

Reply:
https://github.com/aspnet/AspNetCore-Tooling/pull/17